Source/WebCore/rendering/updating/RenderTreeUpdaterViewTransition.cpp

```diff
@@ -153,6 +158,69 @@ void RenderTreeUpdater::ViewTransition::buildPseudoElementGroup(const AtomString
 
     if (viewTransitionGroup)
         m_updater.m_builder.attach(*documentElementRenderer.view().viewTransitionRoot(), WTFMove(viewTransitionGroup), beforeChild);
+
+    auto& resolver = document->styleScope().resolver();
```

Comment:
Is this whole block directly linked to render tree construction? I would prefer putting this code after `setHasViewTransitionPseudoElementTree(true);` in `ViewTransition::setupTransitionPseudoElements()` if not. Otherwise this code will re-run when the group gets `display: none;` removed from its style.